doc/apps/dash.md

```diff
@@ -104,6 +109,27 @@ ploomber-cloud deploy --watch
 To ensure your app doesn't break on re-deployments, pin your [dependencies.](pin-dependencies)
 ```
 
+(customize-deployment)=
+## Customize deployment
+
+To customize the Python version or base image for deployment, you may use [Docker deployment](./docker.md).
+Here's a sample `Dockerfile` for deploying the application:
+
+```text
```

Comment:
no need to put the example dockerfile, just link to the example

Reply:
[Fixed](https://github.com/ploomber/doc/pull/196/files#diff-232e0474ab7eb173493b1cbce28404de549efcd15343850ed95d1bfdd27720baR21-R22)